sphericart-jax/python/tests/test_consistency.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import jax
+import sphericart.jax
+jax.config.update("jax_platform_name", "cpu")
+jax.config.update("jax_enable_x64", True)
+import jax.numpy as jnp
+import numpy as np
+
+
+@pytest.fixture
+def xyz():
+    key = jax.random.PRNGKey(0)
+    return 6 * jax.random.normal(key, (100, 3))
+
+
+def test_consistency(xyz):
+    xyz = jnp.array(xyz, dtype=jnp.float64)
```

Comment:
why is this re-wraped in a `jnp.array`? Should already be one

Reply:
True, it's gone